packages/table/src/components/Form/FormRender.tsx

```diff
@@ -410,11 +410,15 @@ const FormRender = <T, U = any>({
   );
   return (
     <div
-      className={classNames(className, {
-        [formClassName]: isForm,
-        [getPrefixCls(`pro-table-search-${competentName}`)]: true,
-        [`${getPrefixCls('card')}-bordered`]: !!bordered,
-      })}
+      className={classNames(
+        className,
+        {
+          [formClassName]: isForm,
+          [getPrefixCls(`pro-table-search-${competentName}`)]: true,
+          [`${getPrefixCls('card')}-bordered`]: !!bordered,
+        },
+        searchConfig !== false && searchConfig?.className ? searchConfig.className : '',
```

Comment:
```suggestion
        [searchConfig.className]: searchConfig !== false && searchConfig?.className ,
```

既然有 classNames， 就用 classNames 的写法，方便很多

Reply:
有想过，主要怕被后面加的覆盖 😂

Reply:
没事的，一般不搞 class